lib/fluent/plugin/out_kafka_buffered.rb

```diff
@@ -197,9 +208,20 @@ def configure(conf)
       end
     end
 
-    @monitoring_list.each { |m|
```

Comment:
keep this code for backword compatibility.
With `deprecated`, code should work.